Fix: updated Docker deployment docs (issue #3411)

akshith30388 opened the pull request:
This PR updates the Docker deployment documentation to:
- Replace deprecated UI_WEBSOCKET with WEBSOCKET_HOST and WEBSOCKET_PORT
- Add a “Changing Language in UI” section
- Add a “Next Steps” section for clarity

Fixes: #3411

da-Kai reviewed (commented):
@akshith30388,
you can revert anything before your `Changing the Language in the UI` section. 
I added comments, as to why my adjustments to the docs in #3391 differ from yours.

akshith30388 commented:
Hi @da-Kai — thanks for spotting that!

The change that removed the leading = from the document title was unintended (it affects the AsciiDoc heading). I’ve reverted that single line so the title is back to:

= Deploy OpenEMS Edge to Docker


All your other suggestions have been applied and the file has been updated to resolve the conflicts. Could you please re-check and approve the workflow when you have a moment? Thanks a lot for the review and the guidance.

da-Kai commented on doc/modules/ROOT/pages/edge/deploy/docker.adoc:
```diff
@@ -150,3 +151,18 @@ CONTAINER ID  IMAGE                   COMMAND   CREATED               STATUS
+
+== Changing the Language in the UI
+
+To change the language in the OpenEMS UI, open the UI in your browser, click on the settings (⚙️) icon in the top-right corner, and select your preferred language.
```
@akshith30388
I don't know how i missed that, in the previous reviews 😅. I like the idea, but the steps described are wrong. There ist no `settings (⚙️) icon in the top-right corner`

Could you update the instructions to reflect the actual steps to change the UI language?

Sn0w3y replied:
Just to point this out - some AI is hallucinating on this here 😂

akshith30388 commented:
Hi @da-Kai, thanks for the clarifications!

I’ve updated the section based on your feedback:

- Corrected the avatar icon position to **top-left**
- Updated the dropdown label to **“Sprache wählen:” / “Select language:”**
- Removed the reload note, since the language resets on reload (as you mentioned)
- Cleaned up the steps to reflect the actual UI behaviour

Please let me know if anything else should be adjusted. Thanks again for your detailed review!
